src/main/webapp/app/entities/participation-submission/participation-submission-delete-dialog.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { JhiEventManager } from 'ng-jhipster';
+
+import { ExerciseType } from '../exercise';
+
+import { Subscription } from 'rxjs/Subscription';
+import { Submission } from 'app/entities/submission';
+import { ParticipationSubmissionPopupService } from 'app/entities/participation-submission/participation-submission-popup.service';
+import { SubmissionService } from 'app/entities/submission/submission.service';
+
+@Component({
+    selector: 'jhi-participation-submission-delete-dialog',
+    templateUrl: './participation-submission-delete-dialog.component.html',
+})
+export class ParticipationSubmissionDeleteDialogComponent implements OnInit {
+    // make constants available to html for comparison
```

Comment:
Where do you use those? I think this is unused and can be removed.

Reply:
Hi @jpbernius ,
We are using this component for the popup dialog to confirm the deletion of the submission.

Reply:
Yes, I am talking about the constants below.